components/navbar.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Logo from './../public/mcssLogo.svg'
+import Image from 'next/image'
+
+export const NavBar = (): React.ReactElement => {
```

Comment:
Hmm, the type annotations look a bit odd to me. You probably want to do something like this: 
```
import { FC } from "react";

const NavBar: FC = ({
  title,
  content,
}) => {
  body ...
}

export default NavBar;
```

Sidenote, the convention for our project will be to create the const component first then export default at the bottom.

Reply:
I'm going to follow the type annotation here, but not really sure if that even matters in small components like this which requires no props.

Noted on `export default` convention.